components/connector-service/cmd/connectorservice/connectorservice.go

```diff
@@ -61,10 +68,16 @@ func main() {
 		log.Info(internalSrv.ListenAndServe())
 	}()
 
+	http.Handle("/metrics", promhttp.Handler())
+
+	go func() {
+		log.Fatal(http.ListenAndServe(":9090", nil))
```

Comment:
Fatal is not suitable here as it quits application by calling os.Exit(). In case we both internal and external API is sound but metrics server failed we shouldn't kill the application. Consider using Info.